plugins/additional/additional.cpp

```diff
@@ -2046,6 +2136,17 @@ class RibbonPageComponent : public ComponentBase
 
 		return rbpage;
 	}
+
+	/*
```

Comment:
Why add this code comment?

Reply:
These components contain commented event handler pushes so i added commented event handler pops to stay in sync.